tools/docker/builder/openwrt/Dockerfile

```diff
@@ -67,39 +67,39 @@ WORKDIR /home/openwrt/openwrt
 COPY --chown=openwrt:openwrt profiles_feeds/ /home/openwrt/openwrt/profiles_feeds
 
 RUN mkdir -p files/etc \
-#   We need to keep the hashes in the firmware, to later know if an upgrade is needed:
+    #   We need to keep the hashes in the firmware, to later know if an upgrade is needed:
```

Comment:
I think this was done to allow the continuation backslash to work.However, apparently it's not needed... At least not in CI and not on my own machine. @rmelotte please confirm.

Reply:
I had the impression that the '#' character had to be at column 1. 
Apparently it's not the case anymore since 2015 :neutral_face:
https://github.com/docker/cli/commit/869e08a12bf3bf614e74af34f46a6c1d3d061afb